Gemfile

```diff
@@ -5,7 +5,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby "3.1.2"
 
-gem "react_on_rails", "14.0.0"
+gem "react_on_rails", path: '../react_on_rails'
```

Comment:
You have this change locally only

Reply:
it's just a demo to prove 1508, and it depends on the branch `[react_on_hotwired](theforestvn88:react_over_hotwired)` which has not been merged so it linked to the locally `build` of that branch.

anyway, i don't think this PR should be merged so we don't need to be serious about this.